te_markdown/testdrivers.md

```diff
@@ -1,13 +1,29 @@
 ---
 layout: page
-title: Multiple Test Drivers
+title: Test Your Application
 permalink: /te_markdown/testdrivers/
 ---
 
- Fixtures (link)
+* Estimated reading time: 8 minutes
+* Audience: [Curious party], Noob, Somewhat experienced, Experienced, Expert
 
-  this text is on the next page
-  Web-Fixture (html5, angular): Allows you to test contemporary web applications with firefox or chrome
-  Rest-Fixture: Allows to test rest services using json data to drive them
-  Swing-Fixture: Allows to test java swing applications
-  Host-Fixture (mainframe): Allows to test mainframe programs through 3270 terminal emulation
+User acceptance tests should focus on the needs and expected behaviors of the software under test from a user-centric perspective, expressed in technology-agnostic terms of the application domain. However, to automate these tests, the particular technology stack of the software under test have to be taken into account. Effective user acceptance test tools have to cater to both requirements, while also being generic enough to be applicable to the testing of different applications build on diverse platforms and technologies.
+
+**Test-Editor-Web** decouples test specification and test case descriptions from concrete, executable tests that can be run against a given software under test. From test cases expressed with the test case language (TCL), the actual test code is generated on demand, relying on different *test fixtures* that encapsulate the mapping of generic test steps to concrete actions appropriate for a particular technology. Out of the box, the following fixtures are available:
+
+ * Web Fixture: Allows to test contemporary *web applications* (HTML5 in general, with particular support for *Angular* applications) with Firefox or Chrome.
+ * REST Fixture: Allows to test *REST services* using JSON data to drive them.
+ * Swing Fixture: Allows to test *Java Swing* applications.
+ * Host Fixture: Allows to test mainframe programs through *3270 terminal emulation*.
```

Comment:
```suggestion
 * Host Fixture: Allows to test mainframe programs through open source project [X3270](http://x3270.bgp.nu/)
```

Reply:
I would prefer to keep the original text, and have it in italics, for consistency with the other entries in the list. We can add the link, if you like, though I'm not sure what the particular importance is in this case…(?)